config.js

```diff
@@ -6,28 +6,17 @@ var LIB_DEST_PATH = config.LIB_DEST_PATH;
 
 var localConfig = extend(true, {
     bowerConfig: {
-        // Bower configuration for which files to get, and where to put them.
-        // [Source, excluding bower_components]: [Destination].
-        // 'isotope/dist/isotope.pkgd.js': config.LIB_DEST_PATH,
+        'document-register-element/build/document-register-element.max.js': config.LIB_DEST_PATH,
```

Comment:
Turns out I don't know how requireJS works, looks like shim is needed.

Perhaps I should update `marketplace-gulp` to parse the `bowerConfig` and setup the rewrite rules. Seems doable...